### PR TITLE
fix(test): fix project overview acceptance test

### PR DESCRIPTION
**Pull request by billyvg**
fixes timing issue with percy snapshots (sometimes "New Issues" column is still loading)

### CODE REVIEW

**Comment by MaxBittker** [comment]
how about `self.browser.wait_until_not('.loading')`?

**Comment by billyvg** [comment]
ooh, yeah that'll be better. thanks!

**Review by MaxBittker — APPROVED** [review]
🐹

**Review by MaxBittker — APPROVED** [review]
:grimace: this is better than it being broken, but would it be possible to not mock these inline in the components? for example in the config store at least for the footer one? If you considered this and feel this is the least bad option, i'm fine with merging it as-is

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by billyvg** [comment]
@MaxBittker not sure how I wrote that code 🙃

File: src/sentry/static/sentry/app/components/footer.jsx [hunk 1]
```diff
@@ -34,7 +35,9 @@ const Footer = React.createClass({
               <a className="hidden-xs" href="/out/">{t('Migrate to SaaS')}</a>}
           </div>
           <div className="version pull-left">
-            {'Sentry'} {config.version.current}
+            {'Sentry'}
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/components/footer.jsx]: you lost a space, here, could probably fix it with `'Sentry '`

File: docs-ui/components/dynamicWrapper.stories.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withInfo} from '@storybook/addon-info';
+// import {action} from '@storybook/addon-actions';
```
Comment (MaxBittker) [inline docs-ui/components/dynamicWrapper.stories.js]: commented code

**Comment by MaxBittker** [comment]
I like this solution better! hopefully we don't have to sprinkle too many of these around

**Pull request merged**